### PR TITLE
FOUR-9501: Create control LIST TABLE

### PR DESCRIPTION
**Pull request by CarliPinell**
## Issue & Reproduction Steps

Actual behavior: 
A new Control called List Table should be listed in Screen builder. This Control has only 3 options to be selected from dropdown menu. When an option changesm new control List Table changes its header name and its content with a simple vuetable

Expected Behavior:
An official component table and filters are going to be created with the related ticket improvement FOUR-9512.

## Related Tickets & Packages
- [FOUR-9501](https://processmaker.atlassian.net/browse/FOUR-9501)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

## Enviroment
ci:deploy
ci:next
ci:processmaker:feature/FOUR-9486

[FOUR-9501]: https://processmaker.atlassian.net/browse/FOUR-9501?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/renderer/index.js [hunk 1]
```diff
@@ -4,3 +4,4 @@ export { default as FormRecordList } from './form-record-list';
 export { default as FormRecordListStatic } from './form-record-list-static.vue';
 export { default as FormText } from './form-text';
 export { default as FormNestedScreen } from './form-nested-screen';
+export { default as FormListTable } from './form-list-table';
```
Comment (fagubla) [inline src/components/renderer/index.js:7]: add extra line

**Review by fagubla — CHANGES_REQUESTED** [review]

File: src/components/renderer/index.js (hunk 1 above)
Comment (CarliPinell) [reply src/components/renderer/index.js:7]: solved.

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-4b4b8d4356.eng.processmaker.net

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-4b4b8d4356.eng.processmaker.net

**Pull request merged**